### PR TITLE
Pass the parameter connection when register doctrine post persister listener

### LINKED ISSUE #35: Probable fail with other connection name than "default"
See https://github.com/biig-io/DomainComponent/blob/master/src/Integration/Symfony/DependencyInjection/DomainExtension.php#L81

Luckily nobody had this problem until now 😇

### PR DESCRIPTION
**Pull request by morinceline**
Fix #35

### CODE REVIEW

File: src/Integration/Symfony/DependencyInjection/DomainExtension.php [hunk 1]
```diff
@@ -78,7 +78,7 @@ private function registerDoctrinePostPersistListener(array $config, ContainerBui
                     DoctrinePostPersistListener::class
                 )
                 ->setArgument(0, new Reference('biig_domain.dispatcher'))
-                ->addTag('doctrine.event_subscriber', array('connection' => 'default'))
+                ->addTag('doctrine.event_subscriber', array('connection' => $connection))
```
Comment (Nek-) [inline src/Integration/Symfony/DependencyInjection/DomainExtension.php]: Can you add the related test please?

Comment (morinceline) [reply src/Integration/Symfony/DependencyInjection/DomainExtension.php]: yes ;)

File: Tests/Symfony/DependencyInjection/DomainExtensionTest.php [hunk 2]
```diff
@@ -14,14 +14,15 @@ public function testItAddsDoctrinePostPersistListenerToContainer()
...
-                'doctrine' => ['default'],
+                'doctrine' => ['default', 'custom_doctrine'],
             ],
         ]];
 
         $container = new ContainerBuilder();
         $extension->load($config, $container);
 
         $this->assertTrue($container->hasDefinition('biig_domain.post_persist_listener.doctrine_default'));
+        $this->assertTrue($container->hasDefinition('biig_domain.post_persist_listener.doctrine_custom_doctrine'));
```
Comment (Nek-) [inline Tests/Symfony/DependencyInjection/DomainExtensionTest.php]: It does not test the tags (which is what you fixed)

File: Tests/Symfony/DependencyInjection/DomainExtensionTest.php [hunk 3]
```diff
@@ -14,14 +14,31 @@ public function testItAddsDoctrinePostPersistListenerToContainer()
...
+                [
+                    "connection" => "default"
+                ]
+            ],
+            "biig_domain.post_persist_listener.doctrine_custom_doctrine" => [
+                [
+                    "connection" => "custom_doctrine"
+                ]
+            ]
+
```
Comment (Nek-) [inline Tests/Symfony/DependencyInjection/DomainExtensionTest.php]: please remove this space :) . And all is ok! (you can execute php-cs-fixer here https://github.com/biig-io/DomainComponent/blob/master/.php_cs.dist )

**Comment by Nek-** [comment]
Thank you @morinceline !

**Pull request merged**